src/main/java/com/spanish_inquisition/battleship/server/game_states/GameState.java

```diff
@@ -1,11 +1,8 @@
 package com.spanish_inquisition.battleship.server.game_states;
 
 import com.spanish_inquisition.battleship.server.MessageBus;
-import com.spanish_inquisition.battleship.server.Player;
 import com.spanish_inquisition.battleship.server.Players;
 
-import java.util.List;
-
 public abstract class GameState {
     protected Players players;
     protected MessageBus requestBus;
```

Comment:
Dont name a field requestSomething it's a bit misleading (reminds me method name)